packages/core/elasticsearch/core-elasticsearch-server-internal/src/elasticsearch_service.ts

```diff
@@ -155,11 +160,21 @@ export class ElasticsearchService
             'Refer to https://www.elastic.co/guide/en/elasticsearch/reference/master/modules-scripting-security.html for more info.'
         );
       }
+
+      capabilities = getElasticsearchCapabilities({
+        clusterInfo: await firstValueFrom(this.clusterInfo$!),
+      });
```

Comment:
Re-using the `clusterInfo$` observable to retrieve the info required to build the capabilities, which avoids to perform an additional call to ES.